MOONCLOCK/font.py

```diff
@@ -15,6 +15,9 @@
 CHAR_WIDESATOSHI = '\ue011'
 CHAR_MONEY_BAG = '\ue012'
 CHAR_HALVING = '\ue013'
+CHAR_WIFI = '\ue014'
+CHAR_CHECK = '\ue015'
+CHAR_CROSS = '\ue016'
```

Comment:
That doesn't seem to be the correct hexadecimal counting :smile:

7, 8, 9, a, b, c, d, e, f, 11, 12, ....